x-pack/test_serverless/functional/config.base.ts

```diff
@@ -41,6 +41,10 @@ export function createTestConfig(options: CreateTestConfigOptions) {
           `--xpack.trigger_actions_ui.enableExperimental=${JSON.stringify([
             'isUsingRuleCreateFlyout',
           ])}`,
+          `--plugin-path=${resolve(
+            __dirname,
+            '../../../test/analytics/plugins/analytics_ftr_helpers'
+          )}`,
```

Comment:
Can you help me understand what this plugin is doing?
Asking because if this is something-test-only where the behavior is different to what we have in a real serverless environment, then tests based on this would fail when running against MKI. In this case, this config entry should move to a separate config (see e.g. [here](https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/functional/test_suites/observability/config.examples.ts)  how it's done for the example plugins), which doesn't run against MKI.
If it simulates MKI behavior locally, then it can stay. Would probably be good to add a comment explaining it in that case.

Reply:
@elastic/kibana-core could confirm for sure since they maintain the service, but I'm pretty sure this does alter the behaviour because it adds a custom analytics shipper to capture EBT events in tests. I added a separate config for the telemetry tests here: https://github.com/elastic/kibana/pull/201310/commits/67a5954455d8429accd70f139cf86eaecacaa646.

I also wasn't really sure how to handle the custom services to make sure they aren't available where they won't work, so I tried to do something similar to what was done for API integration tests: https://github.com/elastic/kibana/pull/201310/commits/38b930638f469ff0fde2757e44b8d783696bc5a4. Let me know if you think there's a better way to do this and I can update.